src/ray/core_worker/core_worker.cc

```diff
@@ -834,10 +820,6 @@ Status CoreWorker::SubmitActorTask(const ActorID &actor_id, const RayFunction &f
   // Add one for actor cursor object id for tasks.
   const int num_returns = task_options.num_returns + 1;
```

Comment:
@edoakes This +1 was for the actor dummy object. Since we're all going direct, can this be removed as well?

Reply:
I believe @stephanie-wang is using this in the actor handle reference counting now.

Reply:
@stephanie-wang Would you mind explain a bit more on how the +1 on return value is used for ref counting? thanks.